recipes/h5r/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = 'h5r' %}
+{% set version = 'v1.4.7' %}
+
+package:
+  name: r-{{ name }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://cran.r-project.org/src/contrib/Archive/h5r/h5r_1.4.7.tar.gz
+  md5: cba9662fa0bbe9ce0b81c497ec9e09c3
+
+build:
+  number: 0
+  skip: true  # [win]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - r-base
+    - hdf5
+  run:
+    - r-base
+    - hdf5
+
+test:
+    commands:
+      - $R -e "library('h5r')"  # [not win]
+
+about:
+    home: http://r-forge.r-project.org/projects/h5r/
+    summary: r package to provide a simple read/write interface to HDF 5 files
+    license: LGPL
```

Comment:
What is the version of LGPL? Please use the short version in https://spdx.org/licenses/

Reply:
Thanks for asking! The version for LGPL is not explicitly mentioned in the home website or in the package.
http://r-forge.r-project.org/projects/h5r/

Reply:
You also have to include the `license_file:` entry here too.  This is required for LGPL

Reply:
Looks like the license file is not even available in the source folder. With many issues for the package (https://github.com/conda-forge/staged-recipes/pull/8017#issuecomment-480035211), I will pause on making this package available in conda-forge. Thanks for the comments!